content/en/cosign/overview.md

```diff
@@ -63,23 +63,71 @@ $ cosign verify-blob <file> --bundle cosign.bundle --certificate-identity=name@e
 
 To verify, Cosign queries the transparency log (Rekor) to compare the public key bound to the certificate, and checks the timestamp on the signature against the artifact’s entry in the transparency log. The signature is valid if its timestamp falls within the small window of time that the key pair and certificate issued by the certificate authority were valid.
 
-### Working with containers
+## Working with containers
 
-Signing and verifying a container is similar to working with blobs. The Cosign command to sign a container image is:
+> NOTE: You will need access to a container registry for cosign to work with. [ttl.sh](https://ttl.sh/) offers free, short-lived (i.e., hours), anonymous container image hosting if you just want to try these commands out.
+
+Using `ttl.sh` and [crane](https://github.com/google/go-containerregistry/tree/main/cmd/crane) to prepare the image that we want to sign. Run the following:
+
+```
+$ SRC_IMAGE=busybox
+$ SRC_DIGEST=$(crane digest busybox)
+$ IMAGE_URI=ttl.sh/$(uuidgen | head -c 8 | tr 'A-Z' 'a-z')
+$ crane cp $SRC_IMAGE@$SRC_DIGEST $IMAGE_URI:1h
+$ IMAGE_URI_DIGEST=$IMAGE_URI@$SRC_DIGEST
+```
+
+### Keyless signing of container
 
 ```
-$ cosign sign <image URI>
+$ cosign sign $IMAGE_URI_DIGEST
```

Comment:
Given this is an overview section, I would revert back to only including `cosign sign <image URI>` without the output. This is already in signing_with_blobs.md and signing_with_containers.md.

Reply:
took out the output.